README.md

```diff
@@ -13,30 +13,8 @@ This is the Android version of the CMS BPHC app. The app uses standard Moodle en
 4. Download course content once; access locally any time.
 5. Notifications for new content

```

Comment:
Let's add dark theme to this.

Reply:
I'll  do that in a different PR? 
Need to remove Token login from features as well since we use Google Login now.

Reply:
I'll do this in a different PR. Closing this review for now.